src/transformers/models/oneformer/backbone_dinat_oneformer.py

```diff
@@ -0,0 +1,603 @@
+# coding=utf-8
```

Comment:
Rather than implementing this new file, one could add the `DinatBackbone` class to `modeling_dinat.py`.

Then, in `modeling_oneformer.py`, the `AutoBackbone` class can be leveraged, to make use of any backbone.

Reply:
@praeclarumjj3 yes, it would be the best if you could wait until the other PR is merged :)

Just to clarify, could you add the `DinatBackbone` class to `src/transformers/models/dinat/modeling_dinat.py`? You can take a look at the ResNetBackbone class example over [here.](https://github.com/huggingface/transformers/blob/9dfd757fa9cc76ba6833e9018cbb5a9abf8f7d95/src/transformers/models/resnet/modeling_resnet.py#L434)

The idea is to load the backbone with `XXXBackbone(backbone_config)` within modeling_oneformer.py.

Reply:
Absolutely, I will soon update the code with a `DinatBackbone` class :)